ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManagerMetrics.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.clustermap;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import java.util.List;
+
+
+/**
+ * Metrics for the {@link HelixClusterManager}
+ */
+class HelixClusterManagerMetrics {
+  private final HelixClusterManager.HelixClusterManagerCallback clusterMapCallback;
+  private final MetricRegistry registry;
+
+  public Counter liveInstanceChangeTriggerCount;
```

Comment:
also consider renaming `Manager` as discussed (can be in a future patch if that is easier).